octo-core/lib/octocore.rb

```diff
@@ -60,10 +65,10 @@ def self.connect_with_config_dir(config_dir)
     self.connect config
   end
 
-  # Loads the true config. The true config is the hierarchial config
-  # @param [Hash] config The base config. Loaded config will be deep merged
-  #   with this
-  # @param [String] file The file from which config should be loaded
+# Loads the true config. The true config is the hierarchial config
+# @param [Hash] config The base config. Loaded config will be deep merged
```

Comment:
I think its your editor. Make sure your editor replaces tab by 2 spaces. Comments should have the same indentation level as the method http://www.rubydoc.info/gems/yard/file/docs/Tags.md#List_of_Available_Tags